installer.sh

```diff
@@ -303,7 +303,16 @@ else
 		set -e
 		echo "[3/3] No prebuilt PHP found, compiling PHP automatically. This might take a while."
 		echo
-		exec "./compile.sh"
+		logical_cpu_count=$([ $(uname) = 'Darwin' ] && sysctl -n hw.logicalcpu_max || nproc --all) #Get available CPUs to pass to compile.sh and speed up compile
```

Comment:
```suggestion
		logical_cpu_count=$([ "$(uname -s)" == "Darwin" ] && sysctl -n hw.logicalcpu_max || nproc --all) #Get available CPUs to pass to compile.sh and speed up compile
```

Note also this might return an empty string if neither of these commands is successful. Not sure what the script will do in such a case.